flo-runner/src/main/java/com/spotify/flo/context/LoggingContext.java

```diff
@@ -51,17 +51,11 @@ static EvalContext composeWith(EvalContext baseContext, Logging logging) {
 
   @Override
   public <T> Value<T> invokeProcessFn(TaskId taskId, Fn<T> processFn) {
-    return super.invokeProcessFn(taskId, () -> {
-      logging.startEval(taskId);
-      final long t0 = System.nanoTime();
-      try {
-        final T tValue = processFn.get();
-        logging.completedValue(taskId, tValue, Duration.ofNanos(System.nanoTime() - t0));
-        return tValue;
-      } catch (Throwable e) {
```

Comment:
the same comment regarding try-catch

Reply:
Same as https://github.com/spotify/flo/pull/178/files/46b674bec5553ff3e1ebc3fdfbb6ef526bbbf201#r224669023